packages/dashboard/src/layout/AppHeader.tsx

```
@@ -47,17 +48,27 @@ export default function AppHeader() {
-    fetch('https://api.github.com/repos/InsForge/InsForge')
-      .then((res) => res.json())
-      .then((data) => {
-        if (data.stargazers_count !== undefined) {
-          setGithubStars(data.stargazers_count);
-        }
-      })
-      .catch((err) => {
-        console.error('Failed to fetch GitHub stars:', err);
-      });
-  }, []);
+  let mounted = true;
```

ESLint

[error] Insert `··`

(prettier/prettier)